.github/workflows/release-js.yml

```diff
@@ -0,0 +1,77 @@
+on:
+  push:
```

Comment:
I thought you wanted to support the `release` trigger. Do you want to release on every merge into master regardless of changes?

Reply:
Updated. The release will only be triggered on commits affecting root `package.json` and there is a step that checks if the version has been changed